src/software/ai/evaluation/robot.h

```diff
@@ -68,3 +68,5 @@ std::optional<bool> robotBeingPassedTo(const BallHistory& ball_states,
 template <typename STATE_TYPE>
 std::optional<STATE_TYPE> findState(boost::circular_buffer<STATE_TYPE> states,
                                     Timestamp& timestamp);
+
+bool pointNearRobot(Point point, Robot robot);
```

Comment:
think its better for this to be `pointNearRobot(const Point &point, const Robot &robot)`

Reply:
resolved